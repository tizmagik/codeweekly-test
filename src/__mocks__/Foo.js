import React from 'react';

export default () => {
  console.log('Mocked foo!');
  return <div>Mocked Foo!</div>;
}
