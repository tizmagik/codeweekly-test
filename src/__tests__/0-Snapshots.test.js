import React from 'react';
import Foo from '../Foo';
import { shallow } from 'enzyme';

describe('Foo Snapshot', () => {
  it('Renders', () => {
    const wrapper = shallow(<Foo />);

    // Assert expected text is renderd
    expect(wrapper.text()).toBe('Foo');

    // Boy, this sounds like a great use case for a Snapshot test!
  });
});
