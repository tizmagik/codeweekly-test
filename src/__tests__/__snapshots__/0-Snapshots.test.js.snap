exports[`Foo Snapshot Renders 1`] = `
<div>
  Foo
</div>
`;
